config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits.yaml

```diff
@@ -49,6 +49,41 @@ presubmits:
         env:
           - name: GINKGO_FOCUS
             value: "Workload cluster creation"
+          - name: GINKGO_SKIP
+            value: "Creating a GPU-enabled cluster"
+          - name: LOCAL_ONLY
+            value: "false"
+        # docker-in-docker needs privileged mode
+        securityContext:
+          privileged: true
+    annotations:
+      testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-azure
+      testgrid-tab-name: pr-e2e
+      testgrid-alert-email: k8s-infra-staging-cluster-api-azure@kubernetes.io
+  - name: pull-cluster-api-provider-azure-e2e-full
```

Comment:
is the plan to add this once we've merged the GPU PR and verified the pull job works?

Reply:
Yes, or the other way around if we prefer. (Although I need to figure out why this failed first.)

The GPU job has passed Prow CI before it was added to `GINKGO_SKIP`, so I know it works in general.

Reply:
Are you rebased off the latest main branch? Could be https://github.com/kubernetes/test-infra/issues/19662

Reply:
No, I accidentally gave both jobs the same `testgrid-tab-name`. Fixed now.